feconf.py

```diff
@@ -582,8 +582,9 @@ def get_empty_ratings():
 # questions.
 MAX_NUMBER_OF_SKILL_IDS = 20
 
-# The prefix for an 'accepted suggestion' commit message.
-COMMIT_MESSAGE_ACCEPTED_SUGGESTION_PREFIX = 'Accepted suggestion by'
+# The commit message prefix template for 'accepted suggestion'.
+COMMIT_MESSAGE_ACCEPTED_SUGGESTION_PREFIX_TEMPLATE = (
```

Comment:
Please add to the documentation what each %s corresponds to since it's really important that the template be filled in correctly.